lib/concurrent/agent.rb

```diff
@@ -102,24 +104,24 @@ def validate(&block)
     alias_method :validate_with, :validate
     alias_method :validates_with, :validate
 
-    # Update the current value with the result of the given block operation,
+    # Update the current value with the result of the given block fast,
     # block should not do blocking calls, use #post_off for blocking calls
     #
-    # @yield the operation to be performed with the current value in order to calculate
+    # @yield the fast to be performed with the current value in order to calculate
     #   the new value
     # @yieldparam [Object] value the current value
     # @yieldreturn [Object] the new value
     # @return [true, nil] nil when no block is given
     def post(&block)
-      post_on(@task_executor, &block)
+      post_on(@io_executor, &block)
```

Comment:
The intent was to preserve the previous behavior, but the name change proved to be more complicated than I thought it would be. I kept getting confused about which of the previous names mapped to the new names! :-) I'll revisit each of the changes.